pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -19,24 +19,40 @@ package scheduling
 import (
 	"fmt"
 	"sort"
+	"strings"
 	"time"
 
 	v1 "k8s.io/api/core/v1"
+	storagev1beta1 "k8s.io/api/storage/v1beta1"
```

Comment:
Will use the the GA version once https://github.com/kubernetes/kubernetes/pull/83474 merges.

Reply:
looks like the referenced PR merged, could you update?

Reply:
Updated

Reply:
NVM, I reverted the changes. It'll require more work, I need to update the object in the scheduler package as well (along with the CSI predicate).